### ISSUE TITLE
Make releases built from "master" include the SHA they were built from

### ISSUE THREAD

**Issue by ryan-williams** [andypetrella/spark-notebook#319]
I just downloaded a `master` release from http://spark-notebook.io and have a file called `spark-notebook-master-scala-2.10.4-spark-1.5.0-hadoop-2.6.0-cdh5.4.2-with-parquet.zip`.

It would be nice if these had the git SHA they were built from rather than "master" in the name.

**Comment by ryan-williams** [comment]
Also, when I `unzip` this file, I get a directory named `spark-notebook-0.6.1-scala-2.10.4-spark-1.5.0-hadoop-2.6.0-cdh5.4.2-with-parquet`, which seems misleading; eventually if I have a real 0.6.1 release, will it conflict with this? Again, git SHA would be clearer.

**Comment by andypetrella** [comment]
Right, the 0.6.1 thingy is wrong on master... the generator didn't pulled the last updates... I need to figure that out. It should be 0.6.2.
But first thing I can do is to have SNAPSHOT in the version so, 0.6.2-SNAPSHOT
And have a file the build that refers the git SHA.

WDYT?

**Comment by ryan-williams** [comment]
Sorry, thought I had responded here, that sounds good if there is a file in the built distro that indicates the SN sha that it was built from. Does such a thing exist today?

**Comment by andypetrella** [comment]
Actually, there is a class that is generated in the cp that contains it, and it is displayed in the notebook UI.

So there isn't technically this SHA file, but you can see it.
If it doesn't seem enough to you, then I can create this file of course

lmkwyt
